utils_cv/action_recognition/dataset.py

```diff
@@ -396,15 +396,17 @@ def init_data_loaders(self) -> None:
             self.train_ds,
             batch_size=self.batch_size * devices,
             shuffle=True,
-            num_workers=db_num_workers(),
+            num_workers=0,
+            # num_workers=db_num_workers(),
             pin_memory=True,
         )
 
         self.test_dl = DataLoader(
             self.test_ds,
             batch_size=self.batch_size * devices,
             shuffle=False,
-            num_workers=db_num_workers(),
+            num_workers=0,
+            # num_workers=db_num_workers(),
```

Comment:
How about setting num_workers = db_num_workers(8) and if that doesn't work then 4, 2, or eventually just 0 as done here? Having 0 workers significantly slows down image reading, which for action recognition likely makes a big difference (it does for image classification)

Reply:
trying with significantly minified data since this is a memory issue